### DISCUSSION TITLE
Test

### DISCUSSION THREAD

**Discussion by HbmMods** [HbmMods/Hbm-s-Nuclear-Tech-GIT#425] [General]
what

**Comment by SoB1nary** [comment]
what

**Comment by Adam29Adam29** [comment]
Idk

**Comment by TheNewGuy19** [comment]
nor do i

**Comment by Vaern** [comment]
why does this have upvotes

**Comment by ncr-t** [comment]
ah yes, github brewed messages  system

**Comment by Joey-Destroyah** [comment]
Me niether

**Comment by ArkhamOficial** [comment]
Constructive Criticism: The new nuclear explosion animation, which is in the debug wand is cool, actually it's the best animation I've ever seen on a nuclear explosion in Minecraft, but two things that had to be added, one is the mushroom glow at the beginning of the explosion, and there also had to be a flash of light, at the beginning of the explosion, just like in this video: https://youtu.be/Y6k_lEr1-Bo

**Reply by HbmMods** [reply]
the system only exists to show off the convection simulation, literally no part of it is final. the shockwave doesn't work properly, the bottom half of the simulation will definitely require more tweaking, the entire animation doesn't even end properly and it doesn't have any of the additional effects (flash, condensation rings, etc)

**Comment by ArkhamHolthaussen** [comment]
Just an observation, in the new version of the mod (1.0.27X4515) the distillation tower is not being able to connect the pipes, I believe it's just a bug in this version, but I just came to warn you and give you feedback.

**Comment by Tribun-2014** [comment]
Такой вопрос, автору этого мода, зачем он удаляет, верни что удалил и пускай игроки сами выбирают что им надо.

**Reply by ntmforever** [reply]
Так надо.

**Reply by loxpopovich228** [reply]
ы

**Comment by SpaceSoldier123** [comment]
Hello

**Comment by loxpopovich228** [comment]
Ё

**Comment by Doglote** [comment]
I love hbm

**Reply by TreeVirus** [reply]
Martin loves hbm more

**Reply by Doglote** [reply]
> Martin loves hbm more

ᅠ ᅠ    ᅠᅠ ᅠ    ᅠ: ᅠ ᅠᅠ  ᅠ ᅠ ᅠᅠᅠᅠᅠ ᅠ ᅠ ᅠᅠᅠ ᅠ ᅠ ᅠᅠᅠ ᅠᅠ ᅠᅠᅠᅠᅠ ᅠ    ᅠ  ᅠ  ᅠ ᅠ ᅠᅠ   ᅠᅠ ᅠᅠᅠᅠ ᅠᅠᅠ ᅠ   ᅠᅠ ᅠᅠᅠᅠ ᅠᅠ ᅠ ᅠ ᅠ ᅠ ᅠᅠ ᅠᅠᅠᅠ ᅠ ᅠ ᅠᅠᅠ  ᅠ ᅠᅠ ᅠ ᅠᅠᅠᅠ ᅠᅠ  ᅠ  ᅠᅠᅠ ᅠᅠᅠ ᅠ ᅠ    ᅠᅠᅠᅠ  ᅠ ᅠᅠ ᅠ ᅠ ᅠᅠᅠᅠᅠᅠ ᅠ  ᅠ ᅠᅠ  ᅠᅠ  ᅠᅠ  ᅠᅠ  ᅠᅠ  ᅠ ᅠᅠᅠᅠ  ᅠᅠ ᅠᅠ ᅠᅠ ᅠ  ᅠ ᅠᅠᅠ ᅠ ᅠ ᅠᅠ  ᅠᅠᅠ ᅠᅠᅠ ᅠ ᅠ ᅠᅠ  ᅠᅠᅠ ᅠᅠᅠ ᅠ ᅠ ᅠ ᅠᅠ  ᅠ ᅠᅠᅠᅠᅠ ᅠ ᅠ ᅠ  ᅠᅠ  ᅠᅠ ᅠ ᅠ ᅠᅠ ᅠᅠᅠᅠ ᅠ ᅠ ᅠᅠᅠ
